src/piano_dataset/piano_tasks.py

```diff
@@ -132,6 +132,55 @@ def prompt_target_split(self, notes_df: pd.DataFrame) -> TargetPromptSplit:
         return target_split
 
 
+class RangeMasking(PianoTask):
```

Comment:
```suggestion
class TimeRangeMasking(PianoTask):
```

Better names please, "RangeMask" is very non-specific.